setup.py

```diff
@@ -15,4 +15,29 @@
 import setuptools
 
 
-setuptools.setup(version="v1.0.0")
+with open("./README.md", "r") as file:
+    long_description = file.read()
```

Comment:
I'd recommend the following instead, it's a little less prone to errors (e.g. I don't think this will work on all platforms as-is).

```suggestion
import pathlib

here = pathlib.Path(__file__).parent.resolve()
long_description = (here / 'README.md').read_text(encoding='utf-8')
```